src/node_perf.h

```diff
@@ -164,13 +165,13 @@ class ELDHistogram : public HandleWrap, public Histogram {
   SET_SELF_SIZE(ELDHistogram)
 
  private:
-  static void DelayIntervalCallback(uv_timer_t* req);
+  static void DelayIntervalCallback(void* user_data);
 
   bool enabled_ = false;
   int32_t resolution_ = 0;
   int64_t exceeds_ = 0;
   uint64_t prev_ = 0;
-  uv_timer_t timer_;
+  TimerWrapHandle timer_;
```

Comment:
Tbh, I would leave this class as-is – `HandleWrap` is a good abstraction for libuv handles, for example it avoids the extra heap allocation that this is introduces, and it is available through `process._getActiveHandles()` and the post-mortem metadata this way…

Reply:
Yeah, I was considering whether or not to just make `TimerWrap` inherit from `HandleWrap`, but doing so makes it a bit more expensive for cases like the QUIC and inspector timers. I'll drop the changes to `ELDHistogram` here. We can always revisit it later if necessary.